test/quantization/fx/test_quantize_fx.py

```diff
@@ -5109,6 +5109,47 @@ def validate_qconfig(qconfig: QConfig, satisfies_constraints: bool):
             weight=FixedQParamsObserver.with_args(scale=1.0, zero_point=0))
         validate_qconfig(qconfig5, satisfies_constraints=False)
 
+    def test_channel_shuffle_lowering(self):
+        # Three versions of channel shuffle
+        class M1(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.op = torch.nn.ChannelShuffle(2)
+
+            def forward(self, x):
+                return self.op(x + x) + x
+
+        class M2(torch.nn.Module):
+            def forward(self, x):
+                return torch.channel_shuffle(x + x, 2) + x
+
+        class M3(torch.nn.Module):
+            def forward(self, x):
+                return torch.nn.functional.channel_shuffle(x + x, 2) + x
+
+        x = torch.randn(4, 4, 4, 4)
+        models = [M1().eval(), M2().eval(), M3().eval()]
+        for i in range(len(models)):
+            m = models[i]
+            m = prepare_fx(m, {"": default_qconfig}, example_inputs=(x,))
+            m_copy = copy.deepcopy(m)
+            m = convert_fx(m)
+            m_ref = convert_to_reference_fx(m_copy)
+            node_occurrence = {
+                ns.call_function(torch.quantize_per_tensor): 1,
+                ns.call_module(torch.nn.ChannelShuffle): 1 if i == 0 else 0,
+                ns.call_function(torch.channel_shuffle): 1 if i > 0 else 0,
```

Comment:
this is a bit confusing, could you move this to the loop, e.g.
```
expected_nodes = [ns.call_module(torch.nn.ChannelShuffle), ns.call_function(torch.channel_shuffle), ...]
for model, node in zip(models, expected_node)
   ...
```

Reply:
Sure, it's done.